bgcval2/analysis_compare.py

```diff
@@ -4204,21 +4211,40 @@ def load_comparison_yml(master_compare_yml_fn):
     details['shifttimes'] = shifttimes            
     details['suites'] = suites
     return details
-    
+
+
+def get_args():
+    """Parse command line arguments."""
+    parser = argparse.ArgumentParser(
+        description=__doc__,
+        formatter_class=argparse.RawDescriptionHelpFormatter)
+    parser.add_argument('-c',
+                        '--config-file',
+                        default=os.path.join(os.getcwd(),
+                                             'bgcval2-config-user.yml'),
+                        help='User configuration file')
+
+    args = parser.parse_args()
+
+    return args
+
 
 def main():
-    if "--help" in argv or len(argv) == 1:
-        print("Running with no arguments. Exiting.")
-        if "--help" in argv:
-            print("Read the documentation.")
-        exit(0)
+    """Run the main routine."""
+    args = get_args()
 
     config_user=None
-    if "bgcval2-config-user.yml" in argv[1:]:
-        config_user = "bgcval2-config-user.yml"
+    if args.config_file:
+        config_user = os.path.join(os.getcwd(), args.config_file)
         print(f"analysis_timeseries: Using user config file {config_user}")
+    else:
+        config_user = os.path.join(os.getcwd(), "bgcval2-config-user.yml")
+        print(f"analysis_timeseries: Using user default file {config_user}")
+    if not os.path.isfile(config_user):
+        print(f"analysis_timeseries: Could not find configuration file {config_user}")
+        sys.exit(1)
 
-    details = load_comparison_yml(argv[1])
+    details = load_comparison_yml(config_user)
```

Comment:
I don't think this is right. `bgcval2-config-user.yml` and the comparison yml are not the same file. 

`bgcval2-config-user.yml` is the one that you wrote, but the input comparison that `load_comparison_yml` wants looks like this: https://github.com/valeriupredoi/bgcval2/blob/main/input_yml/comparison_analysis_template.yml

Reply:
OK can you make the changes then pls?

Reply:
What does `bgcval2-config-user.yml` do though?

Reply:
OK Ive now changed that - the comparison conf file is paramount, and the user config is optional until we manage to plug its functionality